src/include/internal/error_handler.h

```diff
@@ -18,6 +18,11 @@
  */
 #ifndef ERROR_HANDLER_H_
 #define ERROR_HANDLER_H_
+
+/* User defined error codes set greater than unix error codes.*/
+#define INVALID_ETAG		133	/* Invalid ETag */
+#define BAD_DIGEST			134	/* Non-matching HASH */
```

Comment:
#define spacing inconsistent, please check if its aligned in vim.

Reply:
done